src/Renderer/ThreeExtented/FeatureToolBox.js

```diff
@@ -0,0 +1,552 @@
+/**
+ * Generated On: 2016-09-28
+ * Class: FeatureToolBox
+ * Description:
+ */
+
+import * as THREE from 'three';
+import CVML from 'Core/Math/CVML';
```

Comment:
Aren't we supposed to get rid of CVML? #65 

It seems you only use CVML to triangulate, you could use Three's built in [ExtrudeGeometry](https://threejs.org/docs/index.html?q=extru#Reference/Geometries/ExtrudeGeometry) to achieve the same effect. Or use a third-party library such as [earcut](https://www.npmjs.com/package/earcut).